circleci/circleci.go

```diff
@@ -0,0 +1,34 @@
+package circleci
+
+import (
+	"github.com/airware/vili/log"
+	"github.com/jszwedko/go-circleci"
+)
+
+var config *Config
+var client *circleci.Client
+
+// Config is the circle configuration
+type Config struct {
+	Token   string
+	BaseURL string
```

Comment:
It looks like `BaseURL` isn't actually used

Reply:
We need the BaseUrl to initialize the circle ci client

Reply:
It looks like the `BaseURL` field is optional in the [circleci client](https://godoc.org/github.com/jszwedko/go-circleci#Client), defaulting to the standard API endpoint. We aren't currently providing this value in the client initializer.

Reply:
ok, I'll get rid of it. Thanks